addons/mail/tests/discuss/test_message_controller.py

```diff
@@ -182,52 +201,6 @@ def test_channel_message_attachments(self):
             ],
             "guest should be allowed to add attachment with token when updating message",
         )
-        # test message update: own attachment ok
-        res5 = self.url_open(
-            url="/mail/message/update_content",
-            data=json.dumps(
-                {
-                    "params": {
-                        "message_id": data2["mail.message"][0]["id"],
-                        "body": "test",
-                        "attachment_ids": [self.attachments[1].id],
-                    },
-                }
-            ),
-            headers={"Content-Type": "application/json"},
-        )
-        self.assertEqual(res5.status_code, 200)
-        data3 = res5.json()["result"]
-        self.assertEqual(
-            data3["ir.attachment"],
-            [
-                {
-                    "checksum": False,
-                    "create_date": fields.Datetime.to_string(self.attachments[0].create_date),
-                    "id": self.attachments[0].id,
-                    "name": "File 1",
-                    "res_name": "Test channel",
-                    "mimetype": "application/octet-stream",
-                    "thread": {"id": self.channel.id, "model": "discuss.channel"},
-                    "voice": False,
-                    'type': 'binary',
-                    'url': False,
-                },
-                {
-                    "checksum": False,
-                    "create_date": fields.Datetime.to_string(self.attachments[1].create_date),
-                    "id": self.attachments[1].id,
-                    "name": "File 2",
-                    "res_name": "Test channel",
-                    "mimetype": "application/octet-stream",
-                    "thread": {"id": self.channel.id, "model": "discuss.channel"},
-                    "voice": False,
-                    'type': 'binary',
-                    'url': False,
-                },
-            ],
-            "guest should be allowed to add own attachment without token when updating message",
-        )
```

Comment:
Shouldn't this be replaced by a check that it's not ok to update own message if token is not provided?

Reply:
We have another test [here](https://github.com/odoo/odoo/blob/2ea59156529892ecad5c3ff35a15907b65c03315/addons/mail/tests/discuss/test_message_controller.py#L124) with sending the wrong token. Isn't that kind of the same as not sending the token?